nautobot_ssot/templates/nautobot_ssot/sync_detail.html

```diff
@@ -120,6 +129,77 @@
                     </tr>
                 </table>
             </div>
+            {% if object.load_source_memory_size %}
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <strong>Memory Usage Stats</strong>
+                </div>
+                <table class="table table-hover panel-body attr-table">
+                    <tr>
+                        <td>Memory used loading {{ object.source }}</td>
+                        <td>
+                            <table class="table table-hover panel-body attr-table">
+                                <tr>
+                                    <td>Final</td>
+                                    <td>{{ object.load_source_memory_size | intcomma }} bytes</td>
+                                </tr>
+                                <tr>
+                                    <td>Peak</td>
+                                    <td>{{ object.load_source_memory_size | intcomma }} bytes</td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Memory used loading {{ object.target }}</td>
+                        <td>
+                            <table class="table table-hover panel-body attr-table">
+                                <tr>
+                                    <td>Final</td>
+                                    <td>{{ object.load_target_memory_size | intcomma }} bytes</td>
+                                </tr>
+                                <tr>
+                                    <td>Peak</td>
+                                    <td>{{ object.load_target_memory_peak | intcomma }} bytes</td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>Memory used calculating diff</td>
+                        <td>
+                            <table class="table table-hover panel-body attr-table">
+                                <tr>
+                                    <td>Final</td>
+                                    <td>{{ object.diff_memory_size | intcomma }} bytes</td>
```

Comment:
Would it be more correct to rename the fields to `diff_memory_final` instead of `diff_memory_size`, etc.?